modules/virt-creating-layer2-nad-cli.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc              
+
+:_content-type: PROCEDURE                                             
+[id="virt-creating-layer2-nad-cli_{context}"]                                   
+= Creating a NAD for flat layer 2 topology using the CLI
+
+You can create a network attachment definition (NAD) which describes how to attach a pod to the layer 2 overlay network.
+
+.Prerequisites
+* You have access to the cluster as a user with `cluster-admin` privileges.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+. Create a `NetworkAttachmentDefinition` object. The following example configures a cluster-wide overlay without a subnet defined. This means that the logical switch implementing the network only provides layer 2 communication. You must define a static IP address for the pods.
++
+[source,yaml]
+----
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: l2-network
+  namespace: default
+spec:
+  config: |2
```

Comment:
the user can also configure the MTU of the network. 

Should that (optional) knob be referenced in the example ? 

Keep in mind it must take into account the geneve tunnel overhead.

Reply:
I have added the `mtu` parameter and description. Let me know if that is what you meant by your comment.

Reply:
Hm, the default is not 1300 !!! 

It's chosen by the kernel - but I don't think it is 1300. 

Can we just leave it out in the open w/ `The default value is automatically set by the kernel.` 

set / chosen ?